x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/job_type.tsx

```diff
@@ -22,23 +22,23 @@ export const JobType: FC<Props> = ({ type, setFormState }) => {
     'xpack.ml.dataframe.analytics.create.outlierDetectionHelpText',
     {
       defaultMessage:
-        'Outlier detection jobs require a source index that is mapped as a table-like data structure and analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
+        'Outlier detection identifies unusual data points in the dataset. Outlier detection jobs analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
     }
   );
 
   const regressionHelpText = i18n.translate(
     'xpack.ml.dataframe.analytics.create.outlierRegressionHelpText',
     {
       defaultMessage:
-        'Regression jobs analyze only numeric fields. Use the advanced editor to apply custom options, such as the prediction field name.',
+        'Regression makes predictions on your data after it determines certain relationships among your data points. Regression jobs analyze numeric, categorical, and boolean fields. Use the advanced editor to apply custom options.',
```

Comment:
If the default message of outlier detection is limited to one sentence, then I suggest using only the first sentence here as well.

```suggestion
        'Regression predicts numerical values in the data set. Regression jobs analyze numeric, categorical, and boolean fields. Use the advanced editor to apply custom options.',
```

Reply:
Updated in f3feaac3a3bbf55688e365081e904507aef754bc